src/index.js

```diff
@@ -1 +1,61 @@
 import './style.css';
+
+const gameId = 'lk96JlhTBDrWvcnLFAii';
+const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`;
+
+// fetch(url, {
+//   method: 'POST',
+//   headers: {
+//     'Content-Type': 'application/json'
+//   },
+//   body: JSON.stringify({ name: 'My cool new game' })
+// })
+// .then(response => response.json())
+// .then(data => console.log(data.result));
+
+// const scoreItem = document.getElementById('display-scores');
+// const form = document.getElementById('form');
+// const refresh = document.getElementById('refresh');
+
+const form = document.getElementById('form');
+const refresh = document.getElementById('refresh');
+
+form.addEventListener('submit', async (event) => {
+  event.preventDefault();
+  const formData = new FormData(form);
+  const data = Object.fromEntries(formData);
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+    const result = await response.json();
+    console.log(result);
+  } catch (error) {
+    console.error(error);
+  }
+});
```

Comment:
- [ ] Ouch! this is really close but once user data is submitted it's not reflected on the scoreboard, the refresh button has to be clicked to see it on the scoreboard. Please make sure to fix this. ✔
  ![firefox_JnyS7Jv4ux](https://github.com/nilab-khwaja/Leaderboard/assets/104968236/26299baa-906f-44ab-9422-5b40d89aea38)